core/models/managers.py

```diff
@@ -0,0 +1,35 @@
+"""
+This special file is for keeping track of `models.Manager` classes.
+TODO:
+  If this file grows too large to maintain,
+  create a `managers` folder and split out
+  files based on class being managed.
+"""
+from core.query import only_current_instances
+from django.db import models
+from django.utils import timezone
+
+
+class InstanceActionsManager(models.Manager):
+
+    def get_queryset(self):
+        # Ignores: Terminate, Imaging
+        instance_actions = [
+            'Start', 'Stop',
+            'Resume', 'Suspend',
+            'Shelve', 'Shelve Offload', 'Unshelve',
+            'Reboot', 'Hard Reboot',
+            'Resize', 'Redeploy'
```

Comment:
Curious readers might be interested in [server actions list](http://developer.openstack.org/api-guide/compute/server_concepts.html#server-actions) in the OpenStack docs. 

Notably:
> Shelving a server indicates it will not be needed for some time and may be temporarily removed from the hypervisors. This allows its resources to be freed up for use by someone else.